src/PointwiseFunctions/InitialDataUtilities/Tags/InitialData.hpp

```diff
@@ -0,0 +1,46 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/Serialize.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace InitialDataUtilities {
+namespace OptionTags {
+/*!
+ * \ingroup OptionGroupsGroup
+ * Holds the initial data option in the input file
+ */
+template <typename InitialDataType>
+struct InitialData {
+  static constexpr Options::String help =
+      "Options for initial data of evolution system";
+  using type = std::unique_ptr<InitialDataType>;
+};
+}  // namespace OptionTags
+
+namespace Tags {
+// Base tag for the InitialData struct. Can be used to retrieve the InitialData
+// type object from the cache without having to know its template parameter.
+struct InitialDataBase : db::BaseTag {};
```

Comment:
I believe we don't need the base tag anymore based on the plan we made. You can just use the `InitialData` base class as the type of the simple tag and remove its template parameter.